index.js

```diff
@@ -33,23 +33,23 @@ class ProfileData {
 
      
 
-    passRender(){
+    renderPass() {
         profile.innerHTML += 
              `<div class = "stampContainer">
                 <p class = "passStamp">NOPE</p>
               </div>`
     }
 }
 let doggy = {}
+let index = 0;
 
 function getNewDoggy(){
-    const nextDoggy = dogs[dogsNames.shift()]
+    const nextDoggy = dogs[dogsNames[index % dogsNames.length]];
```

Comment:
the alternative way how to go through an array. 

In that case, there is no need to redefine array values. 
% - is like  7 mod 5 = 2, 8 mod 5 = 3, 10 mod 5 = 0 and etc